src/angularjs/src/app/places/list/places-list.controller.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @ngdoc controller
+ * @name pfb.place.list.controller:PlaceListController
+ *
+ * @description
+ * Controller for listing analysis jobs
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function PlaceListController($state, $stateParams, $scope, Pagination, AuthService,
+                                 Neighborhood, AnalysisJob) {
+        var ctl = this;
+
+        var defaultParams = {
+            limit: null,
+            offset: null,
+            latest: 'True'
+        };
+        var nextParams = {};
+        var prevParams = {};
+
+        initialize();
+
+        function initialize() {
+            ctl.isAdminUser = AuthService.isAdminUser();
+
+            ctl.hasNext = false;
+            ctl.getNext = getNext;
+
+            ctl.hasPrev = false;
+            ctl.getPrev = getPrev;
+            ctl.places = [];
+
+            ctl.filters = {};
+
+            getPlaces();
+        }
+
+        function getPlaces(params) {
+            params = params || $stateParams;
```

Comment:
`defaultParams` needs to be pulled in here or you don't get the `latest=True` (which is probably more obvious on my box, with lots of repeat runs, than on some...).  Perhaps 
```
params = _.merge(defaultParams, $stateParams, params);
```
(Side question: what is `$stateParams` accomplishing here?  Is that to accept params in the page URL and pass them through to the query?)